algviz/docs/list/list.md

```diff
@@ -0,0 +1,13 @@
+# Lists
```

Comment:
Everything list-related should be in its own pull request, but I'll still comment on this one just because it'd be annoying if I withhelf feedback.

Reply:
Ok, thanks. I know its a bit here-and-there, but as we go forward with the project I'll be able to make more logical PRs. I just wanted to add some structure to the project as I was working on it.